databases/week1/homeworkDBw1.sql

```diff
@@ -0,0 +1,55 @@
+-- 1.Find out how many tasks are in the task table
+SELECT COUNT(id) AS 'total_task' FROM task;
+
+
+-- 2.Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*) AS 'total_null' FROM task WHERE due_date IS NULL;
+
+
+-- 3.Find all the tasks that are marked as done
+SELECT * FROM task 
+JOIN status ON task.status_id= status.id
+WHERE status.name LIKE 'done%';
+
+-- 4.Find all the tasks that are not marked as done
```

Comment:
This solution will force you to update your query for each type at status that is added. You only need to narrow down you condition to select those with status not ` 'done'`

Reply:
Perfect. Modify to 
4.Find all the tasks that are not marked as done
```
SELECT * FROM task 
JOIN status ON task.status_id=status.id
WHERE status.name NOT LIKE 'done'
```
Thanks a million!

Reply:
Good job

Reply:
@Lucha2040 Can you please update the code as well? Otherwise the solution will get lost in the comments, plus if other people look over your homework, might miss the right solution as it's hidden in the resolved conversation 🙌

Reply:
Dan, Good you noticed!
It was corrected and merged. 
Thanks!

Reply:
Hi 👍 is the sql updated or it is just me who can not see it?